autotest/gdrivers/bag.py

```diff
@@ -666,49 +665,29 @@ def test_bag_write_south_up():
 
     gdal.Unlink('/vsimem/out.bag')
 
+
 ###############################################################################
 #
 
 
 def test_bag_read_invalid_bag_vlen_bag_version():
-
     if gdaltest.bag_drv is None:
         pytest.skip()
 
     ds = gdal.Open('data/invalid_bag_vlen_bag_version.bag')
     assert not ds
 
-###############################################################################
-# See https://github.com/OSGeo/gdal/issues/1643
-
-
-def test_bag_non_standard_georeferencing_convention():
 
+def test_bag_read_incorrect_northeast_corner():
     if gdaltest.bag_drv is None:
         pytest.skip()
 
-    gdal.Translate('/vsimem/out.bag', 'data/byte.tif',
-                   creationOptions = ['VAR_DATE=2019-06-21'])
-    gdal.Translate('/vsimem/out_non_standard.bag', 'data/byte.tif',
-                   creationOptions = ['VAR_DATE=2019-06-21',
-                                      'CORNER_POINTS_EXTEND_HALF_PIXEL=YES'])
-
-    f = gdal.VSIFOpenL('/vsimem/out.bag', 'rb')
-    assert f
-    data = gdal.VSIFReadL(1, 1000000, f)
-    gdal.VSIFCloseL(f)
-    gdal.Unlink('/vsimem/out.bag')
+    ds = gdal.Open('data/test_offset_ne_corner.bag')
```

Comment:
is that one completely inconsistent, or using a pixelisarea convention ? I'd wish we keep a test for a dataset with a pixelisarea convention

Reply:
We were attempting to add a test case for BAGs that were being assessed as pixelisarea due to the upper right corner being offset to ensure they were being handled as intended.